src/main/kotlin/domain/RacingGame.kt

```diff
@@ -0,0 +1,17 @@
+package domain
+
+import util.RandomNumber
+
+class RacingGame(
+    private var cars: List<Car>
```

Comment:
```suggestion
   val cars: List<Car>
```

List 자체가 불변일텐데 가변일 필요가 있을까요?  val로 만들면 getCars도 제거할 수 있어보여요~